### PR TITLE
docs: Add article about clever-show

### PR DESCRIPTION
**Pull request by goldarte**
Add simple article about [clever-show](https://github.com/CopterExpress/clever-show) project.

### CODE REVIEW

File: docs/en/SUMMARY.md [hunk 1]
```diff
@@ -76,6 +76,7 @@
   * [LED strip (legacy)](leds_old.md)
   * [Contribution Guidelines](contributing.md)
 * Clever-based projects
+  * [Drone show](clever-show.md)
```
Comment (okalachev) [inline docs/en/SUMMARY.md:79]: All the rest article filenames are with underscores.

File: docs/en/clever-show.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+## Demo video
+
+<iframe width="966" height="543" src="https://www.youtube.com/embed/HdHbZFz7nR0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+12 drones perform in a show in Electrotheatre Stanislavsky, Moscow.
+
+## Team
+
+* Arthur Golubtsov, software engineer COEX, https://github.com/goldarte
+* Artem Vasyunik, software engineer intern COEX, https://github.com/artem30801
```
Comment (okalachev) [inline docs/en/clever-show.md:18]: software engineer in COEX, software engineer intern in COEX?

File: docs/ru/clever-show.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# clever-show
+
+Програмное обеспечение для запуска шоу дронов под управлением Raspberry Pi с пакетом COEX [Clever](https://github.com/CopterExpress/clever) и полётного контроллера с прошивкой PX4.
+
+Создайте анимацию в Blender, сконвертируйте её в полётные пути дронов, настройте дроны и запустите своё собственное шоу дронов!
+
+Репозиторий проекта: https://github.com/CopterExpress/clever-show
```
Comment (okalachev) [inline docs/ru/clever-show.md:7]: I would put a dot in the end.

File: docs/en/SUMMARY.md (hunk 1 above)
Comment (goldarte) [reply docs/en/SUMMARY.md:79]: This is the name of the software: "clever-show"

File: docs/en/clever-show.md (hunk 2 above)
Comment (goldarte) [reply docs/en/clever-show.md]: Updated

File: docs/ru/clever-show.md (hunk 3 above)
Comment (goldarte) [reply docs/ru/clever-show.md]: Updated

**Pull request merged**